src/profile.c

```diff
@@ -337,6 +342,26 @@ static void fcloseMaybeCompressed(struct ProfileState* ps)
   ps->Stream = 0;
 }
 
+// When a child is forked off, we force profile information to be stored
+// in a new file for the child, to avoid corruption
+void InformProfilingThatThisIsAForkedGAP(void)
+{
+    HashLock(&profileState);
+    if (profileState_Active) {
+        char filenamecpy[GAP_PATH_MAX + 20];
```

Comment:
It's because otherwise I get a warning from gcc complaining my snprintf could overflow it's buffer (which seems like a stupid warning, as isn't that the point of snprintf?) I couldn't figure out how to disable.

Reply:
However, I accept that's stupid. I've now done it "properly", without having any warnings.